modules/packages/MPI.chpl

```diff
@@ -69,20 +78,26 @@ Note that the same caveats that apply to OpenMP+MPI
 programming apply here. For instance, MPI must be initialized with the
 appropriate thread support (see below).
 
-The second mode of operation is where both Chapel and MPI communications
+Multilocale Mode
+~~~~~~~~~~~~~~~~
+
+Multilocale mode is where both Chapel and MPI communications
 may be mixed. This allows the user to use optimized/convenient MPI routines
 when desired, while having access to the Chapel PGAS model. Using this
 model requires
 
-1. The MPI library should (and in many cases must) support the MPI_THREAD_MULTIPLE
-   support level for threads. This is needed if the user plans on making
-   MPI calls from multiple threads. It is necessary when using the mpi-conduit
-   for GASNet, since the GASNet library may make simultaneous MPI calls.
-   We've had good experience with MPICH and MPI_THREAD_MULTIPLE.
-2. ``CHPL_TASKS=fifo`` is currently a requirement that hopefully can be relaxed
+1. The MPI library should (and in many cases must) support the
+   :const:`MPI_THREAD_MULTIPLE` support level for threads. This is needed if
+   the user plans on making MPI calls from multiple threads. It is necessary
+   when using the mpi-conduit for GASNet, since the GASNet library may make
+   simultaneous MPI calls.  We've had good experience with MPICH and
+   :const:`MPI_THREAD_MULTIPLE`.
+
+2. ``CHPL_TASKS=fifo`` is currently a
+   requirement that hopefully can be relaxed
    in the future.
 
-This second mode has been tried on
+Multilocale mode has been tested on:
 
 1. Linux, with the mpi-conduit for GASNet
 2. Cray XC, with the aries conduit for GASNet
```

Comment:
Let's hedge our bets on this. While I've had this running in some cases, it's been a little hit-or-miss. In part, there are some cases where MPI runs out of memory --- which I think get fixed with hugepage support, but I haven't had the time to fully debug.

The mpi-conduit has been tested both on Linux and Cray systems.